Logo/bin/generate-assets.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
```

Comment:
This should probably be `#!/bin/bash` so that more scripting features are available.

Then, at the top the script should use `set -e` to abort if any errors occur.

Reply:
Done!